get-to-know-security-manager/office-365-groups-screen.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Office 365 Groups Screen
+description: Detailed overview of connected Office 365 Groups.
+author: Petra Filipi
+date: 05/04/2018
+---
+The Office 365 Groups screen gives you a general overview of your connected Office 365 groups, as well as the report for each group in particular.
+
+To start exploring features and reports, navigate to Office 365 Groups and select your tenant.
+
+Available management actions for Office 365 groups are contextually related to the selected object, which means users can __click on or hover over something to get a contextual menu__ that offers actions related to the selected item.
```

Comment:
or hover over the/a group. ne something

Reply:
neću to ispravljati, nije samo grupa u pitanju